api/jobs.yaml

```diff
@@ -130,6 +130,37 @@ paths:
           $ref: '#/responses/ServerError'
       tags:
         - Jobs
+  '/aggregations':
+    get:
+      operationId: GetJobAggregation
+      summary: Query for aggregated jobs in the given time frame
+      parameters:
+        - name: timeFrame
+          description: Time Frame
+          required: true
+          type: string
```

Comment:
This should reference the `TimeFrame` definition below:
```
- name: timeFrame
  description: Time frame to aggregate over
  required: true
  in: query
  schema:
    $ref: '#/definitions/TimeFrame'
```

Reply:
Seems that if we use the TimeFrame schema, we have to put the parameter to request body, is that what we want?

Reply:
I think it may be possible, see "Reusable Enums" here: https://swagger.io/docs/specification/2-0/enums